src/main/scala/com/wavesplatform/network/messages.scala

```diff
@@ -17,7 +17,7 @@ case class GetBlock(signature: ByteStr) extends Message
 case class LocalScoreChanged(newLocalScore: History.BlockchainScore) extends Message
 case class RawBytes(code: Byte, data: Array[Byte]) extends Message
 case class BlockForged(block: Block) extends Message
-case class MicroBlockInv(totalBlockSig: ByteStr, prevBlockSig: ByteStr)  extends Message
+case class MicroBlockInv(totalBlockSig: ByteStr, prevBlockSig: ByteStr, created: Long = System.currentTimeMillis()) extends Message
```

Comment:
No reason to use default values

Reply:
There is no reason to write `System.currentTimeMillis()` everywhere